### PR TITLE
OSSMDOC-158 and OSSMDOC-156

### PR DESCRIPTION
**Pull request by neal-timpe**
https://issues.redhat.com/browse/OSSMDOC-157

### CODE REVIEW

File: modules/ossm-control-plane-profiles.adoc [hunk 1]
```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/prepare-to-deploy-applications-ossm.adoc
+
+[id="ossm-control-plane-profiles_{context}"]
+= Creating control plane profiles
+
+You can create reusable configurations with `ServiceMeshControlPlane` profiles. Individual users can extend the profiles they create with their own configurations. Profiles can also inherit configuration information from other profiles. For example, you can create an accounting control plane for the accounting team and a marketing control plane for the marketing team. If you create a development template and a production template, members of the marketing team and the accounting team can extend the development and production profiles with team specific customization.
+
+When you configure control plane profiles, which follow the same syntax as the `ServiceMeshControlPlane`, users inherit settings in a hierarchical fashion. The Operator is delivered with a `default` template with default settings for {ProductName}.
```
Comment (brian-avery) [inline modules/ossm-control-plane-profiles.adoc:10]: should this be profile? 
The Operator is delivered with a `default` profile with default settings for {ProductName}.

File: modules/ossm-control-plane-profiles.adoc [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                mountPath: /usr/local/share/istio-operator/templates/
+          volumes:
+            - name: smcp-templates
+              configMap:
+                name: smcp-templates
+...
+----
+. Save your changes and exit the editor.
+
+. You can now use the `template` parameter in the `ServiceMeshControlPlane` to specify a template.
```
Comment (brian-avery) [inline modules/ossm-control-plane-profiles.adoc:76]: `profiles` parameter

**Review by brian-avery — APPROVED** [review]

**Review by gbaufake — APPROVED** [review]

**Review by JStickler — CHANGES_REQUESTED** [review]
We probably need to update both v1x and v2x with how the hierarchy is determined.

File: modules/ossm-control-plane-profiles.adoc [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+                mountPath: /usr/local/share/istio-operator/templates/
+          volumes:
+            - name: smcp-templates
+              configMap:
+                name: smcp-templates
+...
+----
+. Save your changes and exit the editor.
+
+. You can use the `proviles` parameter in the `ServiceMeshControlPlane` to specify a template.
```
[the remark this answers is not in the document]
Comment (neal-timpe) [reply modules/ossm-control-plane-profiles.adoc]: It's a list, so it's one or more. Thanks for catching that.

File: modules/ossm-control-plane-profiles.adoc [hunk 4]
```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/prepare-to-deploy-applications-ossm.adoc
+
+[id="ossm-control-plane-profiles_{context}"]
+= Creating control plane profiles
+
+You can create reusable configurations with `ServiceMeshControlPlane` profiles. Individual users can extend the profiles they create with their own configurations. Profiles can also inherit configuration information from other profiles. For example, you can create an accounting control plane for the accounting team and a marketing control plane for the marketing team. If you create a development template and a production template, members of the marketing team and the accounting team can extend the development and production profiles with team specific customization.
+
+When you configure control plane profiles, which follow the same syntax as the `ServiceMeshControlPlane`, users inherit settings in a hierarchical fashion. The Operator is delivered with a `default` profile with default settings for {ProductName}.
```
[the remark this answers is not in the document]
Comment (neal-timpe) [reply modules/ossm-control-plane-profiles.adoc:10]: I agree this should be improved, but not in this PR. It's been in the docs for a while, and I don't think we should hold up the rest of the changes in this PR. Can you create a bug or improvement to capture this comment @JStickler ?

[the remark this answers is not in the document]
Comment (JStickler) [reply modules/ossm-control-plane-profiles.adoc:10]: I created OSSMDOC-163 to capture this.

**Review by JStickler — APPROVED** [review]

**Pull request merged**